### ISSUE TITLE
Allow calculating additional ARO worker nodes in Azure Pricing Calculator

### ISSUE THREAD

**Issue by cwe1ss** [Azure/OpenShift#298]
The Azure Pricing Calculator for ARO always requires 3 master nodes and a minimum of 3 worker nodes. While this is fine for new systems, it is severely limiting when trying to estimate costs for workloads that are added to *existing OpenShift clusters* and therefore just require additional worker nodes. 

I'm therefore asking to please make the **master nodes optional** and allow us to specify a **minimum of 1 worker nodes**. 

You could show a warning that a minimum of 3 is required for new clusters, or make 3 the minimum when master nodes are selected.

This would also make it much easier to calculate prices when using multiple MachineSets. You could then e.g. add one full ARO and another one with just worker nodes.

**Comment by jboutaud** [comment]
Could you explain a little bit more? Azure Red Hat Openshift (ARO) requires a minimum of 3 control plane nodes and 3 worker nodes. If you add more machine set, let set 1 more with 1 instance type in it, then you end up with 3 control plane and 4 worker. 

As you thinking of the use case of what is the delta if the cluster grows?

**Comment by cwe1ss** [comment]
> As you thinking of the use case of what is the delta if the cluster grows?

Yes. I want to be able to calculate the pricing for an application workload that will require adding e.g. 2 worker nodes to an existing ARO cluster.
